views/index.html

```diff
@@ -72,35 +72,26 @@ <h3>Delete issue on <i>apitest</i></h3>
             crossorigin="anonymous"></script>
     <script>
       $(function() {
-        $('#testForm').submit(function(e) {
-          $.ajax({
-            url: '/api/issues/apitest',
-            type: 'post',
-            data: $('#testForm').serialize(),
-            success: function(data) {
-              $('#jsonResult').text(JSON.stringify(data));
-            }
-          });
-          e.preventDefault();
-        });
         $('#testForm2').submit(function(e) {
+          var url = '/api/issues/apitest';
           $.ajax({
-            url: '/api/issues/apitest',
-            type: 'put',
-            data: $('#testForm2').serialize(),
+            type: 'PUT',
+            url: url,
+            data: $(this).serialize(),
             success: function(data) {
-              $('#jsonResult').text(JSON.stringify(data));
+              alert('_id: ' + data._id + '\nassigned_to: ' + data.assigned_to + '\ncreated_by: ' + data.created_by + '\ncreated_on: ' + data.created_on + '\nissue_text: ' + data.issue_text + '\nissue_title: ' + data.issue_title + '\nopen: ' + data.open + '\nproject: ' + data.project + '\nstatus_text: ' + data.status_text + '\nupdated_on: ' + data.updated_on);
```

Comment:
Why is this being turned in to an alert rather than being rendered on the page?

Additionally, the expected return described in the `/learn` user stories is:
> You can send a PUT request to /api/issues/{projectname} with an _id and one or more fields to update. On success, the updated_on field should be updated, and returned should be { result: 'successfully updated', '_id': _id }.

This does not match what the alert is showing.